1_space_odyssey/index.html

```diff
@@ -10,8 +10,7 @@
 </head>
 <body class="fade-in">
     
-    <script src="./js/mannuel-9000/app.js"></script>
-    <!-- include your javascript file here -->
+    <script src="js/therese/app.js"></script>
```

Comment:
please, put my javascript include back into your file, rename your folder to `therese-9000` to follow the naming convention of the project. well done you have committed to an open source project :-)

![https://media.giphy.com/media/hkzSvpZF0KCxW/giphy-downsized.gif](https://media.giphy.com/media/hkzSvpZF0KCxW/giphy-downsized.gif)